src/main/java/net/snowflake/ingest/streaming/internal/OpenChannelResponse.java

```diff
@@ -130,4 +132,13 @@ void setEncryptionKeyId(Long encryptionKeyId) {
   Long getEncryptionKeyId() {
     return this.encryptionKeyId;
   }
+
+  @JsonProperty("stage_location")
+  void setStageLocation(JsonNode stageLocation) {
```

Comment:
Can we use a proper java contract here instead of JsonNode? JsonNode gives us no compile time guarantees on what object was passed in, makes code much harder to read, and makes it easy to introduce corner case bugs where under some specific conditions the json node is constructed differently.